misc/graylog.conf

```diff
@@ -267,12 +267,15 @@ plugin_dir = plugin
 # Default: true
 #elasticsearch_use_expect_continue = true
 
-# Graylog will use multiple indices to store documents in. You can configured the strategy it uses to determine
+# Graylog will use multiple indices to store documents in. You can configure the strategy it uses to determine
 # when to rotate the currently active write index.
-# It supports multiple rotation strategies:
+# It supports multiple rotation strategies, the default being "count":
 #   - "count" of messages per index, use elasticsearch_max_docs_per_index below to configure
 #   - "size" per index, use elasticsearch_max_size_per_index below to configure
-# valid values are "count", "size" and "time", default is "count"
+#   - "time" interval between index rotations, use elasticsearch_max_time_per_index to configure
+# A strategy may be disabled by specifying the optional valid_rotation_strategies list and excluding that strategy.
+valid_rotation_strategies = count,size,time
```

Comment:
Please add a comment in front of it. We usually comment default options. (like you did below with `elasticsearch_max_write_index_age`)

Reply:
fixed